scripts/build/Dockerfile.openj9-ubuntu

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/ibm-semeru-runtimes:open-8-jdk-focal
+FROM docker.io/library/ibm-semeru-runtimes:open-11-jdk-focal
```

Comment:
We should update [Dockerfile.hotspot-alpine](https://github.com/checkpoint-restore/criu/blob/6e11e7f081828edb1bad7e83ea566cf197c2e514/scripts/build/Dockerfile.hotspot-alpine#L1) and [Dockerfile.hotspot-ubuntu](https://github.com/checkpoint-restore/criu/blob/6e11e7f081828edb1bad7e83ea566cf197c2e514/scripts/build/Dockerfile.hotspot-ubuntu#L1) as well.